doc/introduction/importing_circuits.rst

```diff
@@ -0,0 +1,272 @@
+.. role:: html(raw)
+   :format: html
+
+.. _intro_ref_importing_circuits:
+
+Importing circuits
+==================
+
+PennyLane offers the :mod:`~.io` module to import quantum circuits and operations that were
```

Comment:
```suggestion
PennyLane supports importing quantum circuits and operations that were
```
The `io` module is not so user facing here.

Reply:
Good catch! 🎣